src/bun.js/bindings/BunObject+exports.h

```diff
@@ -64,6 +65,9 @@
     macro(spawnSync) \
     macro(which) \
     macro(write) \
+    macro($) \
+    macro(shellParse) \
```

Comment:
are these functions for debugging / internal testing or are these meant to be more public API?